samples/nextjs/package.json

```diff
@@ -69,6 +69,7 @@
     "eslint-plugin-react": "^7.21.5",
     "eslint-plugin-yaml": "^0.2.0",
     "graphql-let": "^0.16.2",
+    "next-transpile-modules": "^7.3.0",
```

Comment:
Unfortunately we'll still have a (unused) dependency for `next-transpile-modules` in the sample. Not sure there's a way to avoid this one?

Reply:
@ambrauer I'm removing it during `jss create` stage